#客户端请求时发送的数字
int32 num1
int32 num2
---
#服务器响应发送到数据
int32 sum